stellargraph/layer/rgcn.py

```diff
@@ -575,11 +575,16 @@ def build(self):
             model output tensor(s) of shape (batch_size, layer_sizes[-1])
 
         """
+        if multiplicity is None:
+            multiplicity = self.multiplicity
 
-        if self.multiplicity == 1:
+        if multiplicity == 1:
             return self._node_model()
 
         else:
             raise NotImplementedError(
                 "Currently only node prediction if supported for RGCN."
             )
+
+    node_model = deprecated_model_function(_node_model, "node_model")
```

Comment:
The `_node_model` method already wasn't public, so I'm not sure this is necessary?

Reply:
nice catch!